spec/expression.dd

```diff
@@ -2773,27 +2773,41 @@ void foo()
                 $(D __vector)
                 $(D function)
                 $(D delegate)
-                $(D const)
-                $(D immutable)
-                $(D inout)
-                $(D shared)
                 $(D module)
                 $(D package)
```

Comment:
what about `return` I had no idea that was a thing.

Reply:
Yes that is documented under the Identifier form, but I it seems to work without an Identifier too. Haven't tested `== super` and `== __parameters` yet.

Reply:
`is(foo == super)` and`is(foo ==  __parameters)` work very differently though, they bind `foo` to the superclass and parameters respectively IIRC

Reply:
> they bind foo to the superclass and parameters respectively IIRC

No, they declare an optional Identifier. I have tested them now, they work without Identifier like `== return` does:
```d
class C {}
pragma(msg, is(C == super)); // true
C c;
pragma(msg, is(c == super)); // false

void foo(int i);
pragma(msg, is(foo ==  __parameters)); // false
pragma(msg, is(typeof(foo) ==  __parameters)); // true
```
I will move these 3 up from the Identifier form.